src/fragments/lib/in-app-messaging/js/customize.mdx

```diff
@@ -0,0 +1,109 @@
+- `ModalMessage` : override default message Modal UI component
+
+<Callout informational>
+  For developer preview, only the BannerMessage and FullScreenMessage UI
+  components are provided by Amplify. CarouselMessage and ModalMessage
+  components must be manually provided to render Carousel and Modal layouts
+</Callout>
+
+## Integrate Custom Style
+
+Custom component styling can be applied to the default In-App Messaging UI components by passing the `style` prop to the `InAppMessagingProvider`, these styles will take precedence over default and In-App message payload style.
+
+```js
+import {
+  InAppMessageDisplay,
+  InAppMessagingProvider
+} from 'aws-amplify-react-native';
+import { Home } from './src/Home';
+
+const myInAppMessageStyle = {
+  BannerMessage: { container: { backgroundColor: 'lightgrey' } },
+  CarouselMessage: { body: { fontSize: 24 } }
+};
+
+const App = () => {
+  return (
+    <InAppMessagingProvider style={myInAppMessageStyle}>
+      <Home />
+      <InAppMessageDisplay />
+    </InAppMessagingProvider>
+  );
+};
+```
+
+The following optional UI component style properties can be per component:
+
+- `body` : text style applied to the message body
+- `closeIconButton` : view style applied to the close button
+- `closeIconColor` : string color value applied to close icon
+- `container` : view style to the primary container of the message component
+- `header` : text style applied to the message header
+- `image` : image style applied to the message image
+- `primaryButton` : view style applied to the primary message button
+- `secondaryButton` : view style applied to the secondary message button
+
+## Advanced UI
+
+In some use cases, you may want to forego the usage of the Amplify default UI handling altogether while still leveraging the Amplify provided In-App Messaging React context and provider component for In-App message context state. This can be achieved by wrapping your application in an `InAppMessagingProvider` and utilizing the `useInAppMessaging` hook to expose the values of the `InAppMessagingContext`.
+
+```js
+import {
+  InAppMessagingProvider,
+  useInAppMessaging
+} from 'aws-amplify-react-native';
+import { Home } from './src/Home';
+
+const MyInAppMessageDisplay = () => {
+  const { inAppMessage } = useInAppMessaging();
+
+  // Do something with inAppMessage
+};
+
+const App = () => {
+  return (
+    <InAppMessagingProvider>
+      <Home />
+      <MyInAppMessageDisplay />
+    </InAppMessagingProvider>
+  );
+};
+```
+
+`useInAppMessaging` exposes the following functions and values of the `InAppMessagingContext`:
+
+- `clearInAppMessage` : removes the current In-App message (if any) from context state
+- `components` : custom UI components passed to the `InAppMessagingProvider`
+- `displayInAppMessage` : manually render a local In-App message
+- `inAppMessage` : current In-App message (if any) loaded in context state
+- `style` : custom style passed to the `InAppMessagingProvider`
```

Comment:
Let's remove the space between the closing back ticks and colons on these lines:

```suggestion
- `clearInAppMessage`: removes the current In-App message (if any) from context state
- `components`: custom UI components passed to the `InAppMessagingProvider`
- `displayInAppMessage`: manually render a local In-App message
- `inAppMessage`: current In-App message (if any) loaded in context state
- `style`: custom style passed to the `InAppMessagingProvider`
```